nifi-bootstrap/pom.xml

```diff
@@ -41,5 +41,12 @@
 			<groupId>org.apache.nifi</groupId>
 			<artifactId>nifi-expression-language</artifactId>
 		</dependency>
+		<dependency>
+          <groupId>com.sun</groupId>
+          <artifactId>tools</artifactId>
+          <version>1.7.0+</version>
+          <scope>system</scope>
+          <systemPath>${java.home}/../lib/tools.jar</systemPath>
```

Comment:
I don't know if others have had this experience, but whenever tools was pulled in as a dependency this way, it tender to be fragile.